projects/storefrontapp-e2e-cypress/cypress/helpers/asm.ts

```diff
@@ -253,6 +253,17 @@ export function asmB2bCustomerLists(): void {
   cy.get('cx-customer-list table').contains('Account');
   cy.get('cx-customer-list button.cx-asm-customer-list-btn-cancel').click();
   cy.get('cx-customer-list').should('not.exist');
+
+  cy.log('--> start emulation by click cart');
+  asm.asmOpenCustomerList();
+  cy.get('cx-customer-list')
+    .find('.cx-btn-cell')
```

Comment:
please check the case: 'checking custom list features', maybe need to add something to fix it

Reply:
Yes, the operation of switching the 'customer list type' dropdown has been added.